app/controllers/api/v1/products_controller.rb

```diff
@@ -0,0 +1,34 @@
+class Api::V1::ProductsController < ApiController
+  def index
+    @products = Product.all
+    render json: @products, status: :ok
+  end
+
+  def show
+    @product = Product.find(params[:id])
+    render json: @product 
+  end
+
+  def create
+    @product = Product.create(product_params)
+    render json: @product
+  end
+
+  def update
+    @product = Product.find(params[:id])
+    @product.update(product_params)
+    render json: @product
+  end
+
+  def destroy
+    @product = Product.all
+    @product = Product.find(params[:id])
+    @product.destroy
+    render json: @product
```

Comment:
why you have used the instance variable @product or @products variable in all action methods. Can't we use products or products simply?

Reply:
simply 'product'  can also be used instead of '@product' as we are not using views for it.